prisma/schema.prisma

```diff
@@ -80,7 +80,7 @@ model User {
   email         String?    @unique
   emailVerified DateTime?
   image         String?
-  admin         Boolean?   @default(false)
+  role          String?        @default("program_administrator")
```

Comment:
This should probably be using prisma enums as opposed to a raw string. See prisma documentation here. https://www.prisma.io/docs/concepts/components/prisma-schema/data-model#defining-enums . Enums will be included as scalar types in the generated client which can be used across the codebase